docs/app-integration-development/19-OAuth-Functionality.md

```diff
@@ -0,0 +1,69 @@
+---
+tags: [app-integration-development]
+---
+
+# OAuth Functionality
+
+## What is OAuth functionality?
+
+PagerDuty Apps can use OAuth functionality to obtain access tokens that allow the application to interact with the PagerDuty [REST API](/api-reference/) on behalf of a user or as the app itself.
+
+For example, an application with its own UI may wish to call the PagerDuty REST API _as a user_ to obtain data specific to that user. An application that receives PagerDuty webhooks or performs other server-side processing may wish to call back to the PagerDuty _as the app_ to obtain more information or update state.
+
+The capabilities and behavior differ based on whether you choose to use the Scoped OAuth or Classic User OAuth functionality.
+
+## OAuth functionality or PagerDuty API key?
+A PagerDuty App with OAuth functionality is the preferred choice for both third-party integrations and long-lived applications that interact with a single PagerDuty Account.
```

Comment:
It's a bit awkward because some OAuth clients can interact with multiple accounts, so it's best not to mention the number of accounts at all?

```suggestion
A PagerDuty App with OAuth functionality is the preferred choice for both third-party integrations and long-lived applications.
```

Reply:
> Are we talking about that fact that Classic User OAuth can interact with multiple PagerDuty Accounts even when it is not published?

I mean more so, classic user oauth clients _can_ be published right now and interact with multiple PagerDuty accounts.